src/renderer/component/viewers/threeViewer/index.jsx

```diff
@@ -187,7 +193,37 @@ class ThreeViewer extends React.PureComponent<Props, State> {
     }
   }
 
-  transformGroup(group) {
+  // Define component types
+  theme: viewerTheme;
+  themes: { dark: viewerTheme, light: viewerTheme };
+  materialColor: string;
+  // Too complex to add a custom type
+  // Posible solution: https://www.npmjs.com/package/@types/three
```

Comment:
Should we use this instead of `any`?
 https://www.npmjs.com/package/@types/three

Reply:
Nice. Definitely

Reply:
Looks like they are `TypeScript` type definitions :disappointed:

Reply:
I can't find anything for flow :stuck_out_tongue:

Reply:
I can't use / import it:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/three/index.d.ts

Reply:
I gotcha. Oh well.